### DISCUSSION TITLE
Filament and Settings profiles across multiple machines?

### DISCUSSION THREAD

**Discussion by buckeyestargazer** [OrcaSlicer/OrcaSlicer#1033] [General]
I've been printing with an X1C for a little while now and also just received a P1P.  I was expecting all of my custom user filament and settings profiles to be available to both printers, but they are not visible when I choose the P1P in Orca slicer.  

For example, I created a PETG profile from the "Generic PETG" profile and have been using it with the X1C.  But that profile is not available for the P1P.  

Is it possible to copy the profiles somehow so I can use them with the P1P, or do I have to set up these profiles all over again on the P1P side?

**Comment by ufoDziner** [comment]
I was just trying to figure out how to do this as well.  Configuring the compatible printers should be easy, yet it's not.  The compatible printers are listed in the .ini file, but it seems that modifying them by hand doesn't import properly.

Also seems to be the case with the "Processes"

**Reply by buckeyestargazer** [reply]
The thing is, there's only a few settings that prevent an X1C profile from being used with the P1P.  Instead of just not making those profiles available to the P1P, it would be better to make it available with a pop-up message saying "some settings are not available for the P1P" or something like that, with a list of those settings that need to be user adjusted.  That would sure make using these profiles a lot easier. 

In the end I basically manually copied all of my profiles to the P1P.  Hopefully if I get more P1Ps in the future those profiles will work with all machines.

**Reply by hliebscher** [reply]
the problem or feature is all settings has a entry witch printer is compatible with ... if you add the second printer manual it works

**Comment by KD7LRJ** [comment]
I use the X1C profiles regularly to generate GCode and send it to my P1P machines. Works fine for me...

There are certainly things that wouldn't work properly, but the lidar on the X1C isn't useful to me (all my machines have textured build plates), I don't print ASA, PA, PC, etc. on my P1P machines, and I hardly ever use the print by object mode (toolhead clearances are different). There might be other things, but I haven't run into them (yet).

**Comment by ufoDziner** [comment]
The idea is that the filament can be set up and used with any machine that the filament is deemed compatible with.  That information is already in the .ini files, it just needs to be included in the interface for operator use.
